logdna/logdna.go

```diff
@@ -33,9 +33,37 @@ import (
     "time"
 
     "github.com/gliderlabs/logspout/router"
-    "github.com/logdna/logspout/logdna/adapter"
+    "github.com/smusali/logspout/logdna/adapter"
```

Comment:
Should it be imported off your fork?

Reply:
yes for now - in the end, once all looks good, will change it back